src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -118,8 +127,6 @@ public Task StartWorkerProcessAsync()
             _languageWorkerProcess.StartProcess();
 
             _state = LanguageWorkerChannelState.Initializing;
-
-            return _workerInitTask.Task;
         }
```

Comment:
return _workerInitTask.Task; is needed to ensure webhost language channel manager waits for the process to start